components/application-broker/internal/nsbroker/facade_legacy_test.go

```diff
@@ -0,0 +1,194 @@
+package nsbroker_test
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
+	scbeta "github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset/typed/servicecatalog/v1beta1"
+	"github.com/kyma-project/kyma/components/application-broker/internal/nsbroker"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	typedCorev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+const (
+	serviceBrokerAPIVersion = "apiextensions.k8s.io/v1beta1"
+)
+
+//go:generate mockery -name=serviceNameProvider -output=automock -outpkg=automock -case=underscore
+type serviceNameProvider interface {
+	GetServiceNameForNsBroker(ns string) string
+}
+
+// legacyFacade is responsible for creation k8s objects for namespaced broker
+// like it was done in production code before a switch to one k8s service for the application broker.
+// The implementation is a copy of old Facade with removed unnecessary code (broker sync, logging)
```

Comment:
Please add a comment that it's used only in tests

Reply:
added comment